src/quicklist.h

```diff
@@ -40,6 +40,10 @@
  * count: 16 bits, max 65536 (max lp bytes is 65k, so max count actually < 32k).
  * encoding: 2 bits, RAW=1, LZF=2.
  * container: 2 bits, PLAIN=1, PACKED=2.
+ * when the container is PACKED, the quicklist node is a listpack.
+ * when the element is large we use PLAIN, which is a string buffer 
+ * that is both a container and an element itself.
+ * 
```

Comment:
I see each line in this list describes a single field, maybe it's nicer not to break that.
would this comment serve the same purpose?
```suggestion
 * container: 2 bits, PLAIN=1 (a single item as char array), PACKED=2 (listpack with multiple items).
```

Reply:
Yes, this is shorter and clearer than mine, thanks for the suggestion!